File: app/src/main/java/zjm/cst/dhu/readattitude/mvp/presenter/MainPresenter.java
package zjm.cst.dhu.readattitude.mvp.presenter;

import javax.inject.Inject;

import zjm.cst.dhu.readattitude.mvp.view.MainContract;

/**
 * Created by zjm on 2017/5/9.
 */

public class MainPresenter implements MainContract.Presenter {
    private MainContract.View myView;

    @Inject
    public MainPresenter() {
    }

    @Override
    public void attachView(MainContract.View BaseView) {
        myView = BaseView;
    }

    @Override
    public void detachView() {

    }
}

File: app/src/main/kotlin/zjm/cst/dhu/readattitude/mvp/presenter/MainPresenter.kt
package zjm.cst.dhu.readattitude.mvp.presenter

import javax.inject.Inject

import zjm.cst.dhu.readattitude.mvp.view.MainContract

/**
 * Created by zjm on 2017/5/9.
 */

class MainPresenter @Inject constructor()
    : MainContract.Presenter {
    private var myView: MainContract.View? = null

    override fun attachView(BaseView: MainContract.View) {
        myView = BaseView
    }

    override fun detachView() {

    }
}
